src/server/game/Entities/Player/Player.cpp

```diff
@@ -18026,8 +18027,10 @@ void Player::_LoadAuras(PreparedQueryResult result, uint32 timediff)
                 continue;
             }
 
+            const ChrRacesEntry* raceEntry = sChrRacesStore.AssertEntry(GetRace());
```

Comment:
you missed const placement, our code style is `[TYPE] [const][*/&]`

Reply:
Oh sorry I missed that!